sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkOperation.scala

```diff
@@ -93,4 +93,16 @@ private[hive] trait SparkOperation extends Operation with Logging {
     case t =>
       throw new IllegalArgumentException(s"Unknown table type is found: $t")
   }
+
+  protected def onError(): PartialFunction[Throwable, Unit] = {
+    case e: Throwable =>
+      logError(s"Error executing get catalogs operation with $statementId", e)
```

Comment:
This error message still refers to "get catalogs" but is logged for every type of operation.

Reply:
nice catch!thanks!